pkg/server/router/operation.go

```diff
@@ -47,15 +48,40 @@ type GetOperationsRequest struct {
 	Parent string `json:"parent"`
 
 	// A standard filter expression conforming to https://google.aip.dev/160.
-	// For example: 'status = done'.
+	// For example: `done = true`.
 	Filter string `json:"filter"`
 }
 
-func (r GetOperationsRequest) ToServiceRequest() operation.GetOperationsRequest {
-	return operation.GetOperationsRequest{
-		Parent: r.Parent,
-		Filter: r.Filter,
+func (r GetOperationsRequest) GetFilter() string {
+	return r.Filter
+}
+
+func (r GetOperationsRequest) ToServiceRequest() (operation.GetOperationsRequest, error) {
+	var opReq operation.GetOperationsRequest
+
+	declarations, err := filtering.NewDeclarations(
+		filtering.DeclareFunction(filtering.FunctionEquals,
+			filtering.NewFunctionOverload(
+				filtering.FunctionOverloadEqualsString, filtering.TypeBool, filtering.TypeBool, filtering.TypeBool)),
+		filtering.DeclareIdent("done", filtering.TypeBool),
```

Comment:
are these the only filter types? done, true, false?

if so - may be good to define them as consts here